utils/src/main/kotlin/Utils.kt

```diff
@@ -381,7 +382,7 @@ fun File.toSafeURI(): URI {
 }
 
 /**
- * Convenience function for [JsonNode] that returns an empty string if [JsonNode.textValue] is called on a null object
+ * Convenience function for [JsonNode] that returns an empty string if [JsonNode.textValue] is called on a null object,
```

Comment:
Nit: Ideally this would have been a separate commit.

Reply:
Done.